sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -255,10 +256,7 @@ private[orc] class OrcOutputWriter(
   }
 }
 
-private[orc] object OrcRelation extends HiveInspectors {
-  // The references of Hive's classes will be minimized.
-  val ORC_COMPRESSION = "orc.compress"
```

Comment:
We have documented `orc.compress` as option explicitly in https://github.com/apache/spark/blob/d8f45408635d4fccac557cb1e877dfe9267fb326/sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala#L523

Now we depends on the configuration name from an external library. But I think the configuration name should not be changed at all. So looks should be fine.

Reply:
Yep. I agree. I don't think Apache ORC changes this in the future since this is a primitive configuration.
BTW, Thank you for pointing this doc. I'll fix some typos here.

Reply:
Oh, right, the `parquet`.